pmahnke/design.ubuntu.com
# Make vanilla great again

## Done
Removed and ignored node_modules and the Gemfile.lock file. Updated the gulp-sass dep. Remove redundant scss files. Update css paths in the header.

## QA
- Run `gulp`
- Check there are no errors
- Open the server address to see the page
